### PR TITLE
add travis shell script

### PR DESCRIPTION
**Pull request by sibsmc**
added travis shell script to support running backend builds separately

### CODE REVIEW

File: .travis.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+case "$1" in
+    server)
+        cd server
+        bundle install
+        bundle exec rspec spec
```
Comment (olleolleolle) [inline .travis.sh]:
```suggestion
        bundle exec rspec
```

RSpec's default directory is `spec`, so it may be easier to not mention it.

Comment (olleolleolle) [reply .travis.sh]: The above is totally not a blocker to merging this!

**Comment by sibsmc** [comment]
Merging even though build fails due to rspec tests out of date. My swagger branch has fixed all tests. So will rebase it once this is merged.

**Comment by olleolleolle** [comment]
Hooray! This is awesome progress!

**Pull request merged**